cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -115,7 +116,7 @@ controller, and serviceaccounts controller.`,
 				os.Exit(1)
 			}
 
-			if err := Run(c.Complete(), wait.NeverStop); err != nil {
```

Comment:
do we want to wire signals for `cloud-controller-manager`?

https://github.com/kubernetes/kubernetes/blob/master/cmd/cloud-controller-manager/app/controllermanager.go#L78